installation_and_upgrade/ibex_install_utils/tasks/server_tasks.py

```diff
@@ -67,6 +67,7 @@
     "windows-x64",
 )
 
+INSTCONFIGS_GIT_URL = "https://control-svcs.isis.cclrc.ac.uk/git/?p=instconfigs/inst.git;a=heads"
```

Comment:
Should we use the per instrument link as in like `https://control-svcs.isis.cclrc.ac.uk/git/?p=instconfigs/inst.git;a=shortlog;h=refs/heads/NDXPEARL` ?